baseplate/frameworks/pyramid/__init__.py

```diff
@@ -81,7 +82,16 @@ def baseplate_tween(request: Request) -> Response:
         else:
             if request.span:
                 request.span.set_tag("http.status_code", response.status_code)
-                response.app_iter = SpanFinishingAppIterWrapper(request.span, response.app_iter)
+                if hasattr(response, "explanation") and response.explanation:
+                    response.app_iter = SpanFinishingAppIterWrapper(
+                        request.span,
+                        chain(
+                            response.app_iter,
+                            [str.encode(response._status), str.encode(response.explanation)],
```

Comment:
why `str.encode(...)` vs `foo.encode()`?

Reply:
It looks like this may be only a test issue, but in this line https://github.com/Pylons/webtest/blob/master/webtest/lint.py#L296
its checking to make sure that everything in the app_iter is a binary type which fails if we don't convert to binary when adding to the app_iter.

We might be able to make it more generic by doing this `[bytes(response._status, 'utf-8'), bytes(response.explanation, 'utf-8')]`